torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -105,7 +105,7 @@ bool isSupported(Node* node) {
       "aten::fmod.Scalar(Tensor self, Scalar other) -> Tensor",
       "aten::fmod.Tensor(Tensor self, Tensor other) -> Tensor",
       "aten::cos(Tensor self) -> Tensor",
-      "aten::sin(Tensor self) -> Tensor",
+      "aten::sin(Tensor self, *, ScalarType? dtype=None) -> Tensor",
```

Comment:
Tensor expression-related changes probably don't have to in this PR, since I don't think change will break them?